business-central-tests/pom.xml

```diff
@@ -158,9 +158,10 @@
               <kie.wb.url>http://${container.host}:${container.port}/${deployable.context}</kie.wb.url>
               <kie.controller.url>ws://${container.host}:${container.port}/${deployable.context}/websocket/controller</kie.controller.url>
               <org.uberfire.nio.git.daemon.port>${cargo.git.port}</org.uberfire.nio.git.daemon.port>
+              <org.uberfire.nio.git.daemon.enabled>true</org.uberfire.nio.git.daemon.enabled>
             </systemPropertyVariables>
             <failIfNoTests>false</failIfNoTests>
-            <argLine>-Xmx1024m -Dfile.encoding=UTF-8</argLine>
+            <argLine>-Xmx1024m -Dfile.encoding=UTF-8 -Dorg.uberfire.nio.git.daemon.enabled=true</argLine>
```

Comment:
for an unknown reason, it only works if I set it here, the above systemProperties doesn't work